lib/src/config/config.rs

```diff
@@ -643,10 +643,10 @@ pub struct ClientConfig {
     #[builder(default)]
     pub validator: Option<ValidatorConfig>,
 
-    /// The optional zkp configuration
+    /// The optional zk prover configuration
     ///
     #[builder(default)]
-    pub zkp: ZKPConfig,
+    pub zkp: ZKProverConfig,
```

Comment:
Isn't it better to remove the `ZKProverConfig.prover_active` and make this optional?
This way if this is present, then we assume the prover is active and we require the keys. Otherwise we assume the prover is inactive.